tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -494,6 +494,51 @@ public function testAlterTableAutoIncrementIntToBigInt(string $from, string $to,
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+
+    public function testGetDetailsWithTwoSameTablesOnDifferentSchemas(): void
+    {
+        $createTableSQL = 'CREATE TABLE migrations(a text);';
+        $this->connection->executeStatement($createTableSQL);
+        $createTableSQL = 'CREATE TABLE "001_test".migrations(a text);';
+        $this->connection->executeStatement($createTableSQL);
+        $this->connection->executeStatement('SET search_path TO "001_test",public');
+
+        $table = new Table('migrations');
+        $table->addColumn('a', 'text');
+
+        try
+        {
+            $databaseTable = $this->schemaManager->listTableDetails('migrations');
+        } catch (\Exception $e) {
```

Comment:
Why is an exception expected here?

Reply:
Needed since when the patch is not applied, there is an exception so I catch it.

Reply:
This test is meant to fail without this patch. Please remove the try/catch.

Reply:
removed the try catch